pandas/__init__.py

```diff
@@ -1,5 +1,11 @@
 # flake8: noqa
 
+import locale
+locale.setlocale(locale.LC_ALL, 'C.UTF-8')
+
+import locale
+locale.setlocale(locale.LC_ALL, 'C.UTF-8')
```

Comment:
why is this needed at all, much less done twice?

Reply:
This was my mistake, I'm not sure why this was added to __init__.  Thank you for pointing it out!